workflows/utilities/wf_read_QC_trim_se.wdl

```diff
@@ -103,12 +103,12 @@ workflow read_QC_trim_se {
     }
   }
   if ("~{workflow_series}" == "theiaprok") {
-    if (call_kraken) {
+    if ((call_kraken) && defined(kraken_db)) {
       call kraken.kraken2_standalone {
         input:
           samplename = samplename,
           read1 = read1,
-          kraken2_db = select_first([kraken_db]),
+          kraken2_db = select_first([kraken_db, "Kraken2 database not defined"]),
```

Comment:
This won't work because it won't ever be reached if kraken_db isn't provided; you'll have to throw that in the select_first in the output section